tests/test_integration.py

```diff
@@ -104,25 +104,25 @@ def table(catalog: Catalog) -> Table:
 
 @pytest.mark.integration
 def test_table_properties(table: Table) -> None:
-    assert table.properties == {}
+    assert table.properties == {'write.parquet.compression-codec': 'zstd'}
 
     with table.transaction() as transaction:
         transaction.set_properties(abc="🤪")
 
-    assert table.properties == {"abc": "🤪"}
+    assert table.properties == {"abc": "🤪", 'write.parquet.compression-codec': 'zstd'}
 
     with table.transaction() as transaction:
         transaction.remove_properties("abc")
 
-    assert table.properties == {}
+    assert table.properties == {'write.parquet.compression-codec': 'zstd'}
```

Comment:
Nit: I suppose we could just define a constant default properties set with the compression codec set, and then just use that in all the tests except the table properties test. The tables property test would be the union of the DEFAULT properties and the "abc" property.

Reply:
Good point @amogh-jahagirdar Updated the code. 

Can't wait for the new dictionary merge code `d1 | d2`